Picker/PickerToggleTrigger.js

```diff
@@ -1,45 +1,53 @@
-import React from 'react';
-import _ from 'lodash';
-import OverlayTrigger from 'rsuite-utils/lib/Overlay/OverlayTrigger';
-import { placementPolyfill } from '../utils';
-import IntlContext from '../IntlProvider/IntlContext';
+import React from "react";
+import _ from "lodash";
+import OverlayTrigger from "rsuite-utils/lib/Overlay/OverlayTrigger";
+import { placementPolyfill } from "../utils";
+import IntlContext from "../IntlProvider/IntlContext";
 
 const PickerToggleTriggerProps = [
-  'onEntered',
-  'onExited',
-  'open',
-  'defaultOpen',
-  'disabled',
-  'onEnter',
-  'onEntering',
-  'onExit',
-  'onExiting',
-  'onHide',
-  'container',
-  'containerPadding',
-  'preventOverflow'
+  "onEntered",
+  "onExited",
+  "open",
+  "defaultOpen",
+  "disabled",
+  "onEnter",
+  "onEntering",
+  "onExit",
+  "onExiting",
+  "onHide",
+  "container",
+  "containerPadding",
+  "preventOverflow"
 ];
 
 const PickerToggleTrigger = React.forwardRef((props, ref) => {
-  const { pickerProps, speaker, trigger = 'click', open, ...rest } = props;
+  const {
+    pickerProps,
+    speaker,
+    trigger = "click",
+    open,
+    animation = false,
+    ...rest
+  } = props;
   const { placement } = pickerProps;
 
   return (
     <IntlContext.Consumer>
-      {(context) => (
+      {context => (
         <OverlayTrigger
           trigger={trigger}
           ref={ref}
           open={open}
           placement={placementPolyfill(placement, context && context.rtl)}
           speaker={speaker}
+          animation={animation}
           {..._.pick(pickerProps, PickerToggleTriggerProps)}
           {...rest}
```

Comment:
I wonder if you would have been able to pass the new `animation` prop through without any change due to this line.

Reply:
Oh, perhaps you weren't able to pass props to this component, and you're just utilizing the default of `false` for now 👍